### ISSUE TITLE
Bug: prefixing breaks keyframes

### ISSUE THREAD

**Issue by inakianduaga** [pazams/postcss-scopify#7]
Haven't checked this deeply but on first look it seems that the prefixing breaks keyframe definitions, for example, for a prefix `_R`,

``` css
@keyframes mdl-spinner__layer-4-fade-in-out {
  from {
    opacity: 0; }
  65% {
    opacity: 0; }
  75% {
    opacity: 0.99; }
  90% {
    opacity: 0.99; }
  100% {
    opacity: 0; } 
}
```

``` css
@-webkit-keyframes mdl-spinner__layer-4-fade-in-out {
  ._R 0%,
  ._R 65%{
     opacity:0
   }
   ._R 75%,
   ._R 90% {
      opacity:.99
   }
   ._R to{
     opacity:0
   }
}
```

which breaks the keyframe definition.

Basically prefixing shouldn't be applied at all on keyframe definitions? (since the actual rule that uses them will be prefixed anyways)

**Comment by pazams** [comment]
Thanks for reporting! 
I'll examine this by this weekend and report back.

**Comment by pazams** [comment]
fixed in version 0.1.7 on npm
@inakianduaga, please update the package and check it out
thanks!

**Comment by inakianduaga** [comment]
@pazams Works like a charm, thanks!
